apps/examples/testcase/tc_main.c

```diff
@@ -159,6 +167,9 @@ static const tash_cmdlist_t tc_cmds[] = {
 #endif
 #ifdef CONFIG_EXAMPLES_TESTCASE_MEDIA_UTC
 	{"media_utc", utc_media_main, TASH_EXECMD_ASYNC},
+#endif
+#ifdef CONFIG_LIBCXX_UTC
+	{"libcxx_utc", utc_libcxx_main, TASH_EXECMD_ASYNC},
```

Comment:
Please add this by Arphabet order.

Reply:
Okay I will reorder

Reply:
Dear @sunghan-chang,

Current list is not in alphabetical order. Where do you want me to place it? ..

Reply:
Oh, in different change, I reordered them, not yet merged. Ok, don't mind my comment.

Reply:
Ok